docs/development/contribution-documentation/documentation-style-guide/README.md

```diff
@@ -111,7 +111,7 @@ The OpenProject documentation should be as clear and easy to understand as possi
 
 - Write in US English with US grammar.  
 
-  
+We aim to use fair language on our language and documentation and choose gender-neutral terms whenever possible. If this is not possible due to the language, e.g. German, we use : to be as inclusive as possible. Please refer to the [fair language guide](../../translate-openproject/fair-language//) for more specific instructions. 
```

Comment:
_"in German, we use :"_ - It might help to have an example, since English-speakers unfamiliar with German might not what it means to use ":". 

I don't know if we want to add an example in French too, where we normally use "·" or ".", for example: 

- étudiant·e·s
- étudiant.e.s

Reply:
@psatyal a good idea with French! I think you could add it directly via translanting this page https://www.openproject-edge.com/fr/docs/development/translate-openproject/fair-language/ ?